packages/design-system/src/components/dialog/stories/index.js

```diff
@@ -70,22 +80,20 @@ export const _default = () => {
           setToggleDialog(!toggleDialog);
         }}
         isOpen={toggleDialog}
-        title={text('title', 'Headline')}
         contentLabel={'Dialog content Label for modal'}
         actions={ActionsNode}
+        {...args}
       >
         <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}>
-          {text(
-            'body text',
-            'Duka din veranda till fest, för en långväga gäst, i landet lagom är bäst.'
-          )}
+          {message}
         </Text>
       </Dialog>
     </InvertedWrapper>
   );
 };
 
-export const With2Actions = () => {
+// eslint-disable-next-line react/prop-types
```

Comment:
💭 @swissspidy @BrittanyIRL @miina would it be beneficial to disable prop types in storybook files? The storybook definitions will have different props (now that controls are being added)